exchange/utils.go

```diff
@@ -18,7 +18,14 @@ import (
 //   1. BidRequest.Imp[].Ext will only contain the "prebid" field and a "bidder" field which has the params for the intended Bidder.
 //   2. Every BidRequest.Imp[] requested Bids from the Bidder who keys it.
 //   3. BidRequest.User.BuyerUID will be set to that Bidder's ID.
-func cleanOpenRTBRequests(ctx context.Context, orig *openrtb.BidRequest, usersyncs IdFetcher, blables map[openrtb_ext.BidderName]*pbsmetrics.AdapterLabels, labels pbsmetrics.Labels, gDPR gdpr.Permissions, usersyncIfAmbiguous bool) (requestsByBidder map[openrtb_ext.BidderName]*openrtb.BidRequest, aliases map[string]string, errs []error) {
+func cleanOpenRTBRequests(ctx context.Context,
+	orig *openrtb.BidRequest,
+	usersyncs IdFetcher,
+	blables map[openrtb_ext.BidderName]*pbsmetrics.AdapterLabels,
```

Comment:
This looks like a typo. Also, following the Go naming convention, I believe this should be `bLabels`. Sorry, I missed this earlier.